### PR TITLE
Java 16

### PR DESCRIPTION
**Pull request by tbradellis**
This PR adds changes to support Java 16 testing

### CODE REVIEW

File: gradle/script/java.gradle [hunk 1]
```diff
@@ -27,16 +27,6 @@ jar {
     includeEmptyDirs false
 }
 
-// JDK7 is required for scala 2.9 and 2.10. The scala gradle plugin does not allow changing the java compiler at runtime,
```
Comment (twcrone) [inline gradle/script/java.gradle:30]: Is is safe to remove this?

File: instrumentation/cassandra-datastax-3.8.0/src/test/java/com/nr/agent/instrumentation/cassandra/CassandraTest.java [hunk 2]
```diff
@@ -49,9 +50,14 @@
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
 
+/*
+ * Gradle hangs indefinitely when these tests run on Java 16
+ *
+ * FIXME capture jmap/jstack thread dumps?
```
Comment (twcrone) [inline instrumentation/cassandra-datastax-3.8.0/src/test/java/com/nr/agent/instrumentation/cassandra/CassandraTest.java:56]: Is this FIXME meant to stay for now?

**Review by twcrone — APPROVED** [review]
LGTM...couple minor comments.

File: gradle/script/java.gradle (hunk 1 above)
Comment (jasonjkeller) [reply gradle/script/java.gradle:30]: Yea, the agent no longer supports Java 7 and won't even start on it.

File: instrumentation/cassandra-datastax-3.8.0/src/test/java/com/nr/agent/instrumentation/cassandra/CassandraTest.java (hunk 2 above)
Comment (jasonjkeller) [reply instrumentation/cassandra-datastax-3.8.0/src/test/java/com/nr/agent/instrumentation/cassandra/CassandraTest.java:56]: I'm ok with removing or leaving it. It's mostly there to spark inspiration in some poor future soul who decides to actually investigate the issue.

**Pull request merged**